mlir/test/lit.cfg.py

```diff
@@ -250,3 +250,6 @@ def have_host_jit_feature_support(feature_name):
 
 if config.run_rocm_tests:
     config.available_features.add("host-supports-amdgpu")
+
+if config.arm_emulator_executable:
+    config.available_features.add("arm-emulator-executable")
```

Comment:
Maybe just call the feature `arm-emulator` so the requires can be `REQUIRES: arm-emulator`?
```suggestion
    config.available_features.add("arm-emulator")
```